Extensions/Effects/crt-pixi-filter.js

```diff
@@ -0,0 +1,13 @@
+gdjs.PixiFiltersTools.registerFilterCreator('Crt', {
+  makePIXIFilter: function() {
+    var crt = new PIXI.filters.CRTFilter();
+
+    return crt;
+  },
+  updateParameter: function(filter, parameterName, value) {
+    if (!['curvature', 'lineWidth', 'lineContrast', 'verticalLine', 'noise', 'noiseSize', 'vignetting', 'vignettingAlpha', 'vignettingBlur', 'seed', 'time'].includes(parameterName)) return;
+
+    if (filter !== 'verticalLine') filter[parameterName] = gdjs.PixiFiltersTools.clampValue(value, 0, 20);
```

Comment:
Same as another comment, put the generic things at the end of a if/else cascade.